### PR TITLE
style update, AY

### PR DESCRIPTION
**Pull request by axyangsio**
style update:

change link colors
remove right panel
set logo links to homepage
remove bottom nav links

### CODE REVIEW

**Comment by DocOtak** [comment]
Why is the secondary nav bar being removed? It makes pages like the submission guide unnavigable: https://learning.cchdo.io/submitting_data/detailed_guide

---
Also, you don't need to sign your commit message with your initials, that is captured by git itself.

File: docs/conf.py [hunk 1]
```diff
@@ -45,14 +45,18 @@
...
         "image_dark": "_static/logo_cchdo.svg",
+        "link": "/",
     },
     "collapse_navigation": True,
     # Add Google Analytics
     "analytics": {
         "google_analytics_id": "G-89QCC25DF0",
-    }
+    },
+    "navigation_with_keys": False,
```
Comment (DocOtak) [inline docs/conf.py]: this defaults to false, why was it added?

**Comment by axyangsio** [comment]
Oh, I didn't know the right panel is being used. I disabled it to hide the show source part.

**Comment by DocOtak** [comment]
Show source was already disabled: https://github.com/cchdo/cchdo-learning-and-guides/commit/3eaa27cf6d6524bc8513e8a6e24954408838fb55

Hiding the right panel that way and the next/previous links make navigating this site quite a pain on a mobile device. Like truly awful.

**Comment by axyangsio** [comment]
OK, you can take any code you think useful, or cancel this PR

**Comment by DocOtak** [comment]
Can you make a new/clean branch and pull request with just the color changes. Those look good I think.

Then make a separate one with the top bar changes (link back to CCHDO main site and the link to the root "/")

**Comment by axyangsio** [comment]
split this into multiple PRs

**Pull request closed without merging**